package-list.json

```diff
@@ -12,13 +12,12 @@
     },
     {
       "version": "0.1.0",
-      "date": "2019-06-18",
+      "date": "2019-09-18",
```

Comment:
What is the basis for this date?   Freeze date?  Is this still correct?

Reply:
at the time I was asking folks to update the date in package list but I overwrite it in publishing the ballot material anyway so it's inconsequential.